docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -9,281 +9,350 @@ priority: 2
 
-Alluxio uses ZooKeeper to achieve master high availability. Alluxio masters use ZooKeeper for leader
-election. Alluxio clients also use ZooKeeper to inquire about the identity and address of the
-current leader.
+### Alluxio download and installation
 
-ZooKeeper must be set up independently
-(see [ZooKeeper Getting Started](http://zookeeper.apache.org/doc/r3.4.5/zookeeperStarted.html)).
+To deploy Alluxio in a cluster, first [download](https://alluxio.org/download) the Alluxio tar file,
+and copy it to every node (master nodes, worker nodes). Extract the tarball to the same path on
+every node.
 
-After ZooKeeper is deployed, note the address and port for configuring Alluxio below.
+### Configuration
 
-#### Shared Filesystem for Journal
+Each Alluxio node (masters and workers) must be configured for HA. On each Alluxio node,
+create the `conf/alluxio-site.properties` configuration file from the template.
 
-Alluxio requires a shared filesystem to store the journal. All masters must be able to read and
-write to the shared filesystem. Only the leader master will be writing to the journal at any given
-time, but all masters read the shared journal to replay the state of Alluxio.
+```bash
+$ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
+```
 
-The shared filesystem must be set up independently from Alluxio, and should already be running
-before Alluxio is started.
+The configuration parameters which must be set are:
+- `alluxio.zookeeper.enabled=true`
+  - This enables the HA mode for the masters, and informs workers that HA mode is enabled.
+- `alluxio.zookeeper.address=<ZOOKEEPER_ADDRESS>`
+  - The ZooKeeper address must be specified when `alluxio.zookeeper.enabled` is enabled.
+  - The HA masters use ZooKeeper for leader election, and the workers use ZooKeeper to discover the leader master.
+  - Multiple ZooKeeper addresses can be specified by delimiting with commas
+  - Examples: `alluxio.zookeeper.address=1.2.3.4:2181`, `alluxio.zookeeper.address=zk1:2181,zk2:2181,zk3:2181`
+- `alluxio.master.hostname=<MASTER_HOSTNAME>`
+  - This must be set to the correct externally visible hostname for each master node.
+  (Workers ignore this parameter when `alluxio.zookeeper.enabled` is enabled) 
+  - Examples: `alluxio.master.hostname=1.2.3.4`, `alluxio.master.hostname=node1.a.com`
+- `alluxio.underfs.address=<STORAGE_URI>`
+  - This is set to the URI of the shared storage system to mount to the Alluxio root. This shared
+  shared storage system must be accessible by all master nodes and all worker nodes.
+  - Examples: `alluxio.underfs.address=hdfs://1.2.3.4:9000/alluxio/root/`, `alluxio.underfs.address=s3a://bucket/dir/` 
+- `alluxio.master.journal.folder=<JOURNAL_URI>`
+  - This is set to the URI of the shared journal location for the Alluxio leader master to write the journal to,
+  and for standby masters replay journal entries from. This shared shared storage system must be
+  accessible by all master nodes.
```

Comment:
it should also be a filesystem, not an object store

Reply:
I added this info to the recommendation to the HA prerequisites.